view/model/mixin.pug

```diff
@@ -0,0 +1,10 @@
+include ../checkbox/mixin.pug
+
+mixin model(type)
+ section.card(class='model')
```

Comment:
We need to remove `card` around the model. Just put model above the cards.

The card is too small to show big model. And we need a big model to show the all details of the color space shape.